modules/oe_multilingual_url_suffix/tests/Kernel/UrlSuffixInstallationTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_multilingual_url_suffix\Kernel;
+
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\language\Plugin\LanguageNegotiation\LanguageNegotiationSelected;
+use Drupal\language\Plugin\LanguageNegotiation\LanguageNegotiationUrl;
+use Drupal\oe_multilingual\Plugin\LanguageNegotiation\LanguageNegotiationAdmin;
+use Drupal\oe_multilingual_url_suffix\Plugin\LanguageNegotiation\LanguageNegotiationUrlSuffix;
+use Drupal\Tests\oe_multilingual\Kernel\InstallationTest;
+
+/**
+ * Class InstallationTest.
+ */
+class UrlSuffixInstallationTest extends InstallationTest {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = ['oe_multilingual_url_suffix'];
```

Comment:
Just curious, since we are testing the installation, should we not call     $this->container->get('module_installer')->install(['oe_multilingual_url_suffix']); in order to test it?

Reply:
I just did with using common practice. Don't remember when we use this construction in KernelTest $this->container->get('module_installer')->install(['oe_multilingual_url_suffix']);
I assume that installing modules inside Kernel tests by little bit different way: \Drupal\KernelTests\KernelTestBase::enableModules